### ISSUE TITLE
parsed[k].text is not a function

### ISSUE THREAD

**Issue by dowanx** [spencermountain/compromise#721]
When trying to parse this phrase

"She has called twice, not the tv"

I get parsed[k].text is not a function:

![image](https://user-images.githubusercontent.com/1731238/75574598-d4430400-5a1b-11ea-8f25-6b7def77627c.png)

I was able to find where in compromise.mjs it is failing:

![image](https://user-images.githubusercontent.com/1731238/75574678-f8064a00-5a1b-11ea-995f-af2d950c7000.png)

I consoled out parsed[k] which showed true.  This must be way it is failing as true can't have .text as a function :)

It looks like somehow true is getting passed into this function which causes it to fail when .text is ran.

**Comment by spencermountain** [comment]
whoa!  thanks Bryan.
I haven't been able to reproduce this yet - which method are you calling?
```js
nlp("She has called twice, not the tv").json()  //this seems to work
```
ya, an error like this should never happen!
thanks

**Comment by dowanx** [comment]
Whoops, I meant to add that :)

nlp("She has called twice, not the tv").verbs().json()


That should trigger the error.  I am on compromise 13.1.0.

Bryan

**Comment by spencermountain** [comment]
yikes!
will fix this asap. asap is tomorrow.
thank you

**Comment by dowanx** [comment]
Awesome thank you!


Bryan

**Comment by spencermountain** [comment]
hey, thank you for the heads-up.
this is fixed now in `13.1.1`.
cheers

**Comment by dowanx** [comment]
Works great!


Bryan
